src/System.Net.Security/src/System/Net/SecureProtocols/NegoState.cs

```diff
@@ -72,6 +69,12 @@ internal static string DefaultPackage
             }
         }
 
+        internal IIdentity GetIdentity()
```

Comment:
Nit: It's a little weird that this method moved up past ValidateCreateContext during the refactoring; but it's not a huge deal.

I was just wondering why we were adding a method... then I saw we weren't.